admin/development.ini

```diff
@@ -0,0 +1,73 @@
+###
+# app configuration
+# https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+###
+
+[app:main]
+use = egg:c2cgeoportal_admin
+
+pyramid.reload_templates = true
+pyramid.debug_authorization = false
+pyramid.debug_notfound = false
+pyramid.debug_routematch = false
+pyramid.default_locale_name = en
+pyramid.includes =
+    pyramid_debugtoolbar
+
+sqlalchemy.url = postgresql://www-data:www-data@localhost/c2cgeoportal
+schema = main
+srid = 3857
```

Comment:
Whant0s this srid?

Reply:
temporary setting to be able to use the model / create the db

Reply:
ok :-)